storage-proofs/porep/src/stacked/vanilla/graph.rs

```diff
@@ -438,241 +438,3 @@ where
     G: Graph<H>,
 {
 }
-
```

Comment:
I'm not sure if these tests should be moved. From my limited understanding the tests in this file are more like unit tests, which I think belong to the file.

As I currently work on generating coverage reports, I think tests with in files should lead to good coverage, tests in the `tests` directory are integration tests that test the bigger picture.

Reply:
I'm ok with whatever makes sense.  This PR is a bit of a mess in that, it might help if we had a list of tests we want moved to sort of narrow down the scope.

Reply:
@vmx While I'm out, feel free to take this over as well :-)